flexmeasures/data/migrations/versions/126d65cbe6b4_asset_audit_log.py

```diff
@@ -0,0 +1,50 @@
+"""asset_audit_log
+
+Revision ID: 126d65cbe6b4
+Revises: 81cbbf42357b
+Create Date: 2024-05-27 15:48:07.399594
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "126d65cbe6b4"
+down_revision = "81cbbf42357b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "asset_audit_log",
```

Comment:
I had expected we could use the existing `auit_log` table, actually.

You find that unpractical due to affected user and account in that, right?

Reply:
Yes it is because of 'affected_user_id' and 'affected_account_id' vs 'affected_asset_id'